app/controllers/super_admin/organisations_controller.rb

```diff
@@ -20,7 +20,7 @@ def service_emails
 
   def show
     @organisation = Organisation.find(params[:id])
-    @team = @organisation.users.order_by_name_and_email
+    @team = @organisation.users.order("#{sort_column} #{sort_direction}").order_by_name_and_email
```

Comment:
does this work for the email field? the second scope seems to order for email again